packages/babel-preset-default/index.js

```diff
@@ -58,7 +58,6 @@ module.exports = function( api ) {
 	return {
 		presets: [ getPresetEnv() ],
 		plugins: [
-			require.resolve( '@babel/plugin-proposal-object-rest-spread' ),
```

Comment:
We enabled all shipped proposals so we no longer need to explicitly include:
- object rest spread
- async generator functions